server/games/game.py

```diff
@@ -30,15 +31,18 @@ class Victory(IntEnum):
     SANDBOX = 3
 
     @staticmethod
-    def from_gpgnet_string(value):
-        if value == "demoralization":
-            return Victory.DEMORALIZATION
-        elif value == "domination":
-            return Victory.DOMINATION
-        elif value == "eradication":
-            return Victory.ERADICATION
-        elif value == "sandbox":
-            return Victory.SANDBOX
+    def from_gpgnet_string(value: str) -> Optional["Victory"]:
```

Comment:
Well no. It's not an `int` because it doesn't make sense to for example add 2 to a `Victory`. Victory is it's own type. Not sure why the original authors decided to make it an `IntEnum` rather than just an `Enum`. I think I'll change that.